sdk/include/aditof/camera.h

```diff
@@ -493,6 +493,13 @@ class SDK_API Camera {
      */
     virtual Status adsd3500GetStatus(int &chipStatus, int &imagerStatus) = 0;
 
+    /**
+     * @brief Enable/disable ccb as master 
+     * @param[out] enable - 1 - disable, 0 - enable
+     * @return Status
+     */
+    virtual Status adsd3500disableCCBM(bool disable) = 0;
```

Comment:
For consistency, you should use capital D (adsd3500d -> adsd3500D)

Reply:
Fixed.